tgui/packages/tgui/interfaces/APC.js

```diff
@@ -0,0 +1,190 @@
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+      <Section title="Power Channels">
+        <LabeledList>
+          {channelArray.map(channel => {
+            const { topicParams } = channel;
+            return (
+              <LabeledList.Item
+                key={channel.title}
+                label={channel.title}
+                buttons={(
+                  <Fragment>
+                    <Box inline mx={2}
+                      color={channel.status >= 2 ? 'good' : 'bad'}>
+                      {channel.status >= 2 ? 'On' : 'Off'}
+                    </Box>
+                    <Button
+                      icon="sync"
+                      content="Auto"
+                      selected={!locked && (
+                        channel.status === 1 || channel.status === 3
+                      )}
+                      disabled={locked}
+                      onClick={() => act('channel', topicParams.auto)} />
+                    <Button
+                      icon="power-off"
+                      content="On"
+                      selected={!locked && channel.status === 2}
+                      disabled={locked}
+                      onClick={() => act('channel', topicParams.on)} />
+                    <Button
+                      icon="times"
+                      content="Off"
+                      selected={!locked && channel.status === 0}
+                      disabled={locked}
+                      onClick={() => act('channel', topicParams.off)} />
+                  </Fragment>
+                )}>
+                {channel.powerLoad}
+              </LabeledList.Item>
+            );
+          })}
+          <LabeledList.Item label="Total Load">
+            <b>{data.totalLoad}</b>
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+      <Section
+        title="Misc"
+        buttons={!!data.siliconUser && (
+          <Fragment>
+            {!!data.malfStatus && (
+              <Button
+                icon={malfStatus.icon}
+                content={malfStatus.content}
+                color="bad"
+                onClick={() => act(malfStatus.action)} />
+            )}
+            <Button
+              icon="lightbulb-o"
+              content="Overload"
+              onClick={() => act('overload')} />
+          </Fragment>
+        )}>
+        <LabeledList.Item
+          label="Cover Lock"
+          buttons={(
+            <Button
+              icon={data.coverLocked ? 'lock' : 'unlock'}
+              content={data.coverLocked ? 'Engaged' : 'Disengaged'}
+              disabled={locked}
+              onClick={() => act('cover')} />
+          )} />
+        <LabeledList.Item
+          label="Night Shift Lighting"
+          buttons={(
+            <Button
+              icon="lightbulb-o"
+              content={data.nightshiftLights ? 'Enabled' : 'Disabled'}
+              onClick={() => act('toggle_nightshift')} />
```

Comment:
For clarity purposes: if something is enabled, the button for it should have the `selected` property so that it's highlighted in green.